src/components/Lobby/GameView/PlayerView/PlayerView.tsx

```diff
@@ -180,7 +180,11 @@ export default function PlayerView(props: PlayerViewProps): JSX.Element {
 								disableDrawingButton={hasLost}
 								onClickDrawingButton={nextDrawing}
 								disableStartVoteButton={hasLost || !props.lobby.game.players.map(e => e.id).includes(playerId)}
-								onClickStartVoteButton={() => setIsStartVoteModalVisible(true)}
+								onClickStartVoteButton={() => {
+									setSelectedPlayer(undefined);
+									setCurrentVote('yes');
+									setIsStartVoteModalVisible(true);
+								}}
```

Comment:
You can also separate complex logic into dedicated function for more readability